assets/RNAseq_report.Rmd

```diff
@@ -1425,7 +1445,9 @@ for (file in contrast_files){
     fname <- tools::file_path_sans_ext(basename(file))
 
     dir.create(paste("pathway_analysis", fname, sep="/"))
-    dir.create(paste("pathway_analysis", fname, "pathway_heatmaps", sep="/"))
+    if (!isNot(params$citest)) {
```

Comment:
Double negative? Can't this be replaced by ` if (params$citest) `?

Reply:
No, since implementing the previous .R scripts in .Rmd, a simple if case sometimes does not work with the Rmd params that are initiated as empty strings (`''`). That is why I had to do this really awkward isNot solution...(I don't like it either :/ )